src/Calculator/Common/KeyboardShortcuManager.cs

```diff
@@ -714,6 +717,29 @@ private static void OnKeyDownHandler(CoreWindow sender, KeyEventArgs args)
                 }
             }
 
+            private static void OnKeyUpHandler(CoreWindow sender, KeyEventArgs args)
+            {
+                s_keyHandlerCount--;
+                if (s_keyHandlerCount == 0 && s_deferredEnableShortcut)
+                {
+                    KeyboardShortcutManager.DisableShortcuts(false);//shortcuts enabled
+                    s_deferredEnableShortcut = false;
+                }             
+
+            }
+
+            public static void TryEnableShortcut()
```

Comment:
We already have `DisableShortcuts()` to disable shortcuts and `HonorShortcuts()` to enable shortcuts. The 's_fDisableShortcuts' decides the value of `s_fHonorShortcuts` when it is set. Creating another `TryEnableShortcut()` is kind of confusing.

`DisableShortcuts()` is only used when the mode property is changed. We can keep using the `DisableShortcuts()` there and add the checking of `s_keyHandlerCount` in this method.